CHANGELOG.md

```diff
@@ -3,9 +3,12 @@
 `servant-hmac-auth` uses [PVP Versioning][1].
 The change log is available [on GitHub][2].
 
-## 0.1.4 - Jan 27, 2023
+## 0.1.5 - Jan 27, 2023
 * Bump dependency upper bounds, allow building with `GHC 9.0`, `9.2` and `9.4`
 
+## 0.1.4 - March 8, 2022
+* [#55](https://github.com/Holmusk/servant-hmac-auth/pull/55):
+
```

Comment:
I believe there's also:

- Servant 0.19 support (https://github.com/Holmusk/servant-hmac-auth/pull/53)
- GHC 8.10.7 compilation (https://github.com/Holmusk/servant-hmac-auth/pull/51) which you authored :slightly_smiling_face: